### DISCUSSION TITLE
Databricks support

### DISCUSSION THREAD

**Discussion by eichert12** [RamiAwar/dataline#339] [Ideas]
I'm using Databricks and would love to have Dataline support.  Is it something you would consider adding?

**Comment by RamiAwar** [comment]
> I'm using Databricks and would love to have Dataline support.  Is it something you would consider adding?

Definitely! Will try to give an estimate of when we can have this this weekend.

First impression is it should be really easy, I remember it's already supported in the library we're using and we just need to test + expose + document it.

**Comment by RamiAwar** [comment]
I think this'll need some custom setup - I see a host (supported) and API key (not supported in the standard approach) is necessary and some other things are optional (warehouse ID and stuff).

Won't be as easy as I expected it to. This is becoming more and more of a need @anthony2261 , having custom requirements per connection type. Maybe we should generalize the connection type now and create custom templates per new connection type that can automatically get rendered when creating a new instance of that connection type. Wdyt?
